config/codeception/codeception.yml

```diff
@@ -5,6 +5,8 @@ paths:
   support: tests/_support
   envs: tests/_envs
 actor_suffix: Tester
+settings:
+  shuffle: true
```

Comment:
what dis? it shuffles the order of the tests?

Reply:
Yeah, so that we ensure we don't have a dependency on order of operations on the tests.

Reply:
https://codeception.com/docs/07-AdvancedUsage#shuffle

Reply:
I use it locally for maximum hilarity.

Reply:
thats cool. i like it.